pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -150,6 +167,19 @@ public IngestionDelayTracker(ServerMetrics serverMetrics, String tableNameWithTy
     _realTimeTableDataManager = realtimeTableDataManager;
     _clock = Clock.systemUTC();
     _isServerReadyToServeQueries = isServerReadyToServeQueries;
+
+    if (realtimeTableDataManager.getInstanceDataManagerConfig() != null
+        && realtimeTableDataManager.getInstanceDataManagerConfig().getConfig() != null) {
+      PinotConfiguration pinotConfiguration = realtimeTableDataManager.getInstanceDataManagerConfig().getConfig();
+      _enableOffsetLagMetric =
+          pinotConfiguration.getProperty("offset.lag.tracking.enable", DEFAULT_ENABLE_OFFSET_LAG_METRIC);
+      _offsetLagUpdateIntervalMs =
```

Comment:
Should there be a minimum and maximum for this?  For example: what happens if someone puts 0ms or 1ms?

Reply:
Added check for min value!